lib/PluralRules.js

```diff
@@ -0,0 +1,78 @@
+const localeData = {
+	mi: {
+		aliases: ['mri', 'mao'],
+		options: {
+			cardinal: {
+				locale: 'mi',
+				pluralCategories : [ 'one', 'other' ],
+				shim: true
+			},
+			ordinal: {
+				locale: 'mi',
+				pluralCategories : [ 'other' ],
+				shim: true
+			},
+		},
+		select(n, ord) {
+			return !ord && n === 1 ? 'one' : 'other';
+		}
+	}
+};
+
+function getCanonicalLocales(locales) {
+	const mappedLocales = [locales].flat().map(locale => {
```

Comment:
This is only needed for Safari on older OS versions (< ~macOS 12, based on sauce testing), which don't map the aliases back to `mi`.